src/EventManager.ts

```diff
@@ -112,7 +119,7 @@ export default class EventManager {
         return this.oneSignalEventEmitter.addListener(eventName, addListenerCallback);
     }
 
-    getFinalPayload(eventName, payload) {
+    getFinalPayload(eventName: string, payload: any) {
```

Comment:
missing return type

Reply:
For now, made return type any. Will need to update after addressing event types.